set.mm

```diff
@@ -400716,6 +400727,35 @@ a multiplicative function (but not completely multiplicative).
 ###############################################################################
 $)
 
+  $( Declare some new variables. $)
+  $v xO $.
+  $v xL $.
+  $v xR $.
+  $v yO $.
+  $v yL $.
+  $v yR $.
+  $v zO $.
+  $v zL $.
+  $v zR $.
+  $( Let ` xO ` be an individual variable. $)
+  vxo $f setvar xO $.
+  $( Let ` xL ` be an individual variable. $)
+  vxl $f setvar xL $.
+  $( Let ` xR ` be an individual variable. $)
+  vxr $f setvar xR $.
+  $( Let ` yO ` be an individual variable. $)
+  vyo $f setvar yO $.
+  $( Let ` yL ` be an individual variable. $)
+  vyl $f setvar yL $.
+  $( Let ` yR ` be an individual variable. $)
+  vyr $f setvar yR $.
+  $( Let ` zO ` be an individual variable. $)
+  vzo $f setvar zO $.
+  $( Let ` yL ` be an individual variable. $)
+  vzl $f setvar zL $.
+  $( Let ` yR ` be an individual variable. $)
+  vzr $f setvar zR $.
+
```

Comment:
Do we really need these new variables? Since they have global scope, new variables must be handled very carefully.

Reply:
There is/was an (undocumented?) convention to use single lower case letters for setvar variables, see also https://groups.google.com/g/metamath/c/2WoVp7jPeDM/m/G5bMi5I8BwAJ.